### PR TITLE
Add HWP Trotter bloqs and costs to notebook.

### PR DESCRIPTION
**Pull request by fdmalone**
Builds on #753

### CODE REVIEW

File: qualtran/bloqs/chemistry/trotter/hubbard/qpe_cost_optimization.ipynb [hunk 1]
```diff
@@ -446,6 +446,59 @@
     "print(rf\"T_{{opt}} = {t_opt:4.3e}\")"
    ]
   },
+  {
+   "cell_type": "markdown",
```
Comment (mpharrigan) [inline qualtran/bloqs/chemistry/trotter/hubbard/qpe_cost_optimization.ipynb:450]: is this notebook unit tested anywhere? please add and make sure to `@pytest.mark.notebook`

Comment (fdmalone) [reply qualtran/bloqs/chemistry/trotter/hubbard/qpe_cost_optimization.ipynb:450]: For some reason I thought the scipts automagically globbed all the notebooks?

Comment (mpharrigan) [reply qualtran/bloqs/chemistry/trotter/hubbard/qpe_cost_optimization.ipynb:450]: the CI will automatically git-glob all the notebooks in the notebooks check, but it's policy :tm: to also include a unit test that will get picked up by e.g. a local launch of `pytest`

Comment (fdmalone) [reply qualtran/bloqs/chemistry/trotter/hubbard/qpe_cost_optimization.ipynb:450]: done

**Review by mpharrigan — APPROVED** [review]

**Pull request merged**